dev/com.ibm.ws.security.wim.adapter.ldap/resources/OSGI-INF/l10n/metatype.properties

```diff
@@ -419,6 +419,11 @@ referral.desc=Specify the behavior for LDAP referrals. The default behavior is t
 referral.ignore=Ignore LDAP referrals.
 referral.follow=Follow LDAP referrals.
 
+derefAliases=Alias dereferencing handling
+derefAliases.desc=Specify whether when looking up an alias entry, the alias is dereferenced so that the object returned is the object pointed to by the alias' DN.
```

Comment:
When  x(?) looks up an alias entry, this property specifies whether the alias is dereferenced, so that the object that is returned is the object that is pointed to by the distinguished name (DN) of the alias. 

- `when looking up an alias entry` - What looks up the alias entry?

Suggested revision would replace elliptical phrase "when looking", with the agent/process that looks up the alias.

`When x(?) looks up an alias entry, specifies whether the alias is dereferenced so that the object that is returned is the object that is pointed to by the alias DN.`

Reply:
Dave and I decided on the following:

```
When an alias entry is encountered, specifies whether the alias is dereferenced so that the object that is returned is the object that is pointed to by the alias DN.
```